local-db/init-pg.sql

```diff
@@ -48,3 +48,43 @@ CREATE TABLE IF NOT EXISTS "active_pin" (
     created_at DATE NOT NULL,
     updated_at DATE
 );
+
+
+-- PERMISSION -------------------------------------------
+-- Table Definition ----------------------------------------------
+
+CREATE TABLE IF NOT EXISTS "permission" (
+    permission_id UUID PRIMARY KEY NOT NULL,
+    permission VARCHAR(50),
+    role VARCHAR(12)
+);
+
+-- LOG -------------------------------------------
+-- Table Definition ----------------------------------------------
+
+CREATE TYPE expiration_reason_type AS ENUM ('OP', 'CC', 'OR', 'CO', 'OT');
+
+CREATE TABLE IF NOT EXISTS "log" (
+    log_id UUID PRIMARY KEY NOT NULL,
+    pin VARCHAR(8) NOT NULL,
+    pid INT NOT NULL,
+    parcel_status parcel_status_type NOT NULL,
+    title_number VARCHAR(11) NOT NULL, 
+    land_title_district VARCHAR(2) NOT NULL,
+    fron_title_number VARCHAR(11),
+    from_land_title_district VARCHAR(2),
+    title_status title_status NOT NULL,
+    key DATE NOT NULL, 
+    expired_at DATE NOT NULL, 
```

Comment:
according the db diagram, expired_at can be null, but not sure if this makes sense